pysatNASA/instruments/methods/cdaweb.py

```diff
@@ -902,9 +972,12 @@ def cdas_list_remote_files(tag='', inst_id='', start=None, stop=None,
     elif start == stop:
         stop = start + dt.timedelta(days=1)
 
-    if isinstance(start, pds._libs.tslibs.timestamps.Timestamp):
-        start = start.tz_localize('utc')
-        stop = stop.tz_localize('utc')
+    # Ensure that valid date types are used.
+    start = pysat.utils.time.filter_datetime_input(start)
+    stop = pysat.utils.time.filter_datetime_input(stop)
+
+    # cdasws needs a time for the stop date.
+    stop += dt.timedelta(seconds=86399)
```

Comment:
What happens with data that has less than a 1 second cadence? I also don't quite get how this interacts with the previous definitions of stop.

Reply:
cdasws uses stop differently from pysat. See https://github.com/pysat/pysatNASA/pull/223